smoketest/graphqloncallassignments_test.go

```diff
@@ -1,32 +1,51 @@
 package smoketest
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
-	"github.com/target/goalert/engine/resolver"
-	"github.com/target/goalert/smoketest/harness"
-	"strings"
+	"html/template"
```

Comment:
should be `text/template`
```suggestion
	"text/template"
```